sale_order_lot_mrp/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Sale Order Lot MRP
+==================
+
+Allow you to attach a lot in manufacturing order
+that was defined by sale_order_lot_generator module
+for all concerned products (Lot Generation Auto field)
+Name of the MO is also based on sale name.
+
+Usage
+=====
+
+To use this module, you need to:
+
```

Comment:
I found a bug when validation a stock picking (after manufacturing products and generating lot number). I'll fix it.